examples/src/python/example/python_distribution/hello/setup_requires/setup.py

```diff
@@ -14,7 +14,7 @@
 # This is for testing purposes so we can assert that setup_requires is functioning
 # correctly (because Pants swallows print statements).
 if os.getenv('PANTS_TEST_SETUP_REQUIRES', ''):
-  output = [str(pycountry)]
+  output = [bytes(pycountry)]
```

Comment:
Assuming similar logic here that you want bytes instead of unicode? 

If you want unicode, I don't know of a Py2 Py3 compatible way of specifying we want unicode because we can't import six or future. `unicode()` is only supported in Py2.